### PR TITLE
[Messenger] Transport not mandatory

### PR DESCRIPTION
**Pull request by kunicmarko20**
After reading the documentation a few days ago I thought that async is the only way this library works, after seeing the discussion today in #general on slack, @sroze explained it. Hopefully, this will help others if they find themselves in the same problem.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Thanks Marko! I agree this is important. Instead of adding a note below, I propose to reword the introduction so we can tell about this to the reader as soon as possible.

File: messenger.rst [hunk 1]
```diff
@@ -100,9 +100,11 @@ Once you've created your handler, you need to register it:
 Transports
 ----------
 
-The communication with queuing systems or third parties is delegated to
-libraries for now. The built-in AMQP transport allows you to communicate with
-most of the AMQP brokers such as RabbitMQ.
+By default, messages are processed as soon as they are dispatched. If you prefer
+to process messages asynchronously, you must configure a transport. These
+transports communicate your application with queuing systems or third parties.
```
Comment (sroze) [inline messenger.rst]:
```patch
- communicate your application with queuing systems
+ communicate with your application via queuing systems
```

**Review by sroze — APPROVED** [review]
Small rewording, otherwise 👌

**Comment by kunicmarko20** [comment]
Thank you @javiereguiluz, it is a lot better now!

**Comment by javiereguiluz** [comment]
Thanks Marko.

**Pull request merged**